src/features/autothread.ts

```diff
@@ -59,7 +59,7 @@ const autoThread: ChannelHandlers = {
       ? await thread.fetchStarterMessage()
       : undefined;
 
-    if (!starter || !guild) {
+    if (!starter || !guild || user.id !== starter.author.id) {
```

Comment:
This is actually desired behavior, we want to allow 2+ members who aren't the author to mark a response as the answer! see #162